creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -238,13 +238,7 @@ async function _handleIssueSyncRequest({
       return { result: 'success_mode_disabled', syncReqsToEnqueue }
     }
 
-    const libs = await initAudiusLibs({
-      enableEthContracts: true,
-      enableContracts: false,
-      enableDiscovery: true,
-      enableIdentity: false,
-      logger
-    })
+    const libs = await initAudiusLibs({ logger })
```

Comment:
fixes `ERROR || Failed to retrieve user from EntityManager` during sync. this was caused by not initting libs with `contracts`, which it needs when URSM is enabled

Reply:
CON-470

Reply:
why remove other configs here instead of just flipping enableContracts to true

Reply:
I tried with only flipping enableContracts, but it needs identity as well. so it needs all the flags which is the default if no params are passed. moving a different PR anyway